Update deDE.lua

bullsei opened the pull request.

funkydude commented:
Are you sure about `L.none = "Kein"` instead of `L.none = "Nichts"` (copied from Blizzard)?

You had some bad timing on this since I just added the new difficulty indicator translation :P

bullsei commented:
Both is possible https://www.dict.cc/englisch-deutsch/none.html 
But i changed it.
I added the difficultyIndicator. I always use the web editor anyways :)
